### ISSUE TITLE
Memory issue

### ISSUE THREAD

**Issue by instantname** [blatinier/pyhunspell#39]
With hunspell 0.5.0, Python 3, Debian:
```
for i in range(1000):
    h = HunSpell('hunspell.dic', 'hunspell.aff')
```
This keeps filling the memory until the system kills the python process. Same issue if `del h` is added in the loop, or if the `h = ...` is embedded in a function.

**Comment by blatinier** [comment]
There is probably a memory leak I admit, but in the first place, why would you create thousands of hunspell dict?
But it clearly looks like a memory leak.

**Comment by instantname** [comment]
@blatinier I can't see of a good reason to do that. I just stumbled across this while copy-pasting some code to a new app at a development stage. However, that is still a bug!

**Comment by blatinier** [comment]
Ok great, I will look a bit into it.

**Comment by blatinier** [comment]
Fixed by #42
